webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/TechniqueAcceptationDatetimeUpdater.scala

```diff
@@ -278,26 +278,24 @@ class TechniqueAcceptationUpdater(
                                         //ignore ? => do nothing
                                         UIO.unit
                                       case Some(t) =>
-                                        //if the technique is system, we must not add it to UserTechniqueLib, the process
-                                        //must be handled by a dedicated action. Only update system techniques!
```

Comment:
so actually, yes, we want to auto-add it, but there's still a subtility, because the category name does not match